src/app/login/login.scss

```diff
@@ -0,0 +1,22 @@
+@import url(https://fonts.googleapis.com/css?family=Atomic+Age);
+#divCenter {
+  padding-bottom: 3%;
+}
+
+img.screenShot {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+
+}
+.developersBottom {
+  background-color: transparent;
+  border-color: transparent;
+}
+.listCenter {
+  text-align: center;
+}
+body,h1 {
```

Comment:
over bootstrap variable for this. http://getbootstrap.com/customize/#less-variables . example is already given in `index.scss`